From f6b5bf454a61ca0090a1a32b798045a807d8ddad Mon Sep 17 00:00:00 2001
From: Enrico Minack <enrico-minack@gmx.de>
Date: Fri, 10 Dec 2010 08:10:55 +0000
Subject: moved to libnotify version 0.7

fixed API change of libnotify 0.7 for gweather and battstat, fixes #632803
---
diff --git a/battstat/battstat_applet.c b/battstat/battstat_applet.c
index 16dbc31..4948529 100644
--- a/battstat/battstat_applet.c
+++ b/battstat/battstat_applet.c
@@ -435,7 +435,7 @@ battery_full_notify (GtkWidget *applet)
 			GTK_ICON_LOOKUP_USE_BUILTIN,
 			NULL);
 	
-	NotifyNotification *n = notify_notification_new (_("Your battery is now fully recharged"), "", /* "battery" */ NULL, applet);
+	NotifyNotification *n = notify_notification_new (_("Your battery is now fully recharged"), "", /* "battery" */ NULL);
 
 	/* XXX: it would be nice to pass this as a named icon */
 	notify_notification_set_icon_from_pixbuf (n, icon);
diff --git a/configure.in b/configure.in
index 781d8ee..32d7878 100644
--- a/configure.in
+++ b/configure.in
@@ -22,7 +22,7 @@ LIBGTOP_REQUIRED=2.11.92
 LIBXKLAVIER_REQUIRED=4.0
 LIBWNCK_REQUIRED=2.9.3
 LIBGNOME_DESKTOP_REQUIRED=2.11.1
-LIBNOTIFY_REQUIRED=0.3.2
+LIBNOTIFY_REQUIRED=0.7
 HAL_REQUIRED=0.5.3
 DBUS_REQUIRED=1.1.2
 DBUS_GLIB_REQUIRED=0.74
diff --git a/gweather/gweather-applet.c b/gweather/gweather-applet.c
index 3768ded..2acad31 100644
--- a/gweather/gweather-applet.c
+++ b/gweather/gweather-applet.c
@@ -471,8 +471,7 @@ update_finish (WeatherInfo *info, gpointer data)
 			 if (icon == NULL)
 				 icon = "stock-unknown";
 	           	 
-			 n = notify_notification_new (message, detail, icon,
-					 gw_applet->container);
+			 n = notify_notification_new (message, detail, icon);
 	
 		   	 notify_notification_show (n, &error);
 			 if (error)
--
cgit v0.8.3.1

